joplin/base/models/official_documents_page.py

```diff
@@ -66,7 +66,8 @@ class OfficialDocumentPageOfficialDocument(Orderable):
     authoring_office = models.CharField(verbose_name="Authoring office of document", max_length=DEFAULT_MAX_LENGTH)
     summary = models.TextField(verbose_name="Document summary")
     name = models.CharField(verbose_name="Name of Document", max_length=DEFAULT_MAX_LENGTH)
-    document = models.ForeignKey(Document, null=True, blank=False, on_delete=models.SET_NULL, related_name='+')
+    document = models.ForeignKey(Document, null=True, on_delete=models.SET_NULL, related_name='+', verbose_name="Document [en]")
+    document_es = models.ForeignKey(Document, blank=True, null=True, on_delete=models.SET_NULL, related_name='+', verbose_name="Document [es]")
 
```

Comment:
the verbose name now matches the translation pattern we have for the other translated fields.